base/src/main/java/org/robojackets/apiary/base/AppEnvironment.kt

```diff
@@ -0,0 +1,20 @@
+package org.robojackets.apiary.base
+
+import android.net.Uri
+
+enum class AppEnvironment(
+    val apiBaseUrl: Uri,
+    val clientId: String,
+    val production: Boolean = true
```

Comment:
What is this for?

Reply:
Might remove for now, I was kinda unsure if I'd want it for something